src/gourmet/plugins/import_export/epub_plugin/epub_exporter.py

```diff
@@ -55,22 +63,27 @@ def __init__(self, outFileName):
         # This adds the field also known as keywords in some programs.
         self.ebook.add_metadata('DC', 'subject', "cooking")
 
-    def addRecipeCssFromFile(self, filename):
+    def addRecipeCssFromFile(self, filename=None):
```

Comment:
Please add type hinting:
```suggestion
    def addRecipeCssFromFile(self, filename: Optional[str] = None):
```

Reply:
Done.